test/addons-napi/test_instanceof/test.js

```diff
@@ -74,7 +74,8 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
   compareToNative(x, MySubClass);
   compareToNative(y, MySubClass);
   compareToNative(x, MyClass);
-  compareToNative(y, MyClass);
+  // TODO: https://github.com/nodejs/abi-stable-node/issues/236
```

Comment:
Does it make sense to add the ChakraCore check rather than commenting these out?